core/src/navigation/LeftNav.html

```diff
@@ -139,6 +139,11 @@
     {/if} {/each}
   </nav>
   {/if}
+  {#if footerText}
+  <span
+    class="lui-sideNavFooter fd-has-type-minus-1"
```

Comment:
```suggestion
    class="lui-sideNavFooter"
```

In case we decide to use variables. Check my comment below for row `281`